shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/uimodel/QuestionUiModel.kt

```diff
@@ -32,3 +35,17 @@ fun Question.toQuestionUiModel(index: Int, totalStep: Int): QuestionUiModel = Qu
     question = this,
     step = "${index + 1}/${totalStep}"
 )
+
+fun List<QuestionUiModel>.mapToQuestionSubmissions(): List<QuestionSubmission> {
```

Comment:
I think we don't need the `map` prefix here.
```suggestion
fun List<QuestionUiModel>.toQuestionSubmissions(): List<QuestionSubmission> {
```

Reply:
solved in 0318aa5